Debug error

<!-- DO NOT DELETE
validate_template=true
template_path=.github/ISSUE_TEMPLATE/bug_report.md
-->
I have this repository as package dependency in my project, after switching in build settings " C++ and Objective-C Interpretability" to C++/Objective-C++ I have an error during debugging 
error: couldn't IRGen expression: Clang importer error
error: /Users/user/Library/Developer/Xcode/DerivedData/ProjectName-eqlgtjjbjxfjgahdtsezsljspzsa/SourcePackages/checkouts/GoogleUtilities/GoogleUtilities/Environment/Public/GoogleUtilities/GULSecureCoding.h:28:47: error: expected identifier; 'class' is a keyword in Objective-C++
+ (nullable id)unarchivedObjectOfClass:(Class)class
so what can I do with that?

--- a/GoogleUtilities/Environment/GULSecureCoding.m
+++ b/GoogleUtilities/Environment/GULSecureCoding.m
@@ -53,10 +53,10 @@ NSString *const kGULSecureCodingError = @"GULSecureCodingError";
   return object;
 }
 
-+ (nullable id)unarchivedObjectOfClass:(Class)class
++ (nullable id)unarchivedObjectOfClass:(Class)aClass
                               fromData:(NSData *)data
                                  error:(NSError **)outError {
-  return [self unarchivedObjectOfClasses:[NSSet setWithObject:class] fromData:data error:outError];
+  return [self unarchivedObjectOfClasses:[NSSet setWithObject:aClass] fromData:data error:outError];
 }
 
 + (nullable NSData *)archivedDataWithRootObject:(id<NSCoding>)object error:(NSError **)outError {

--- a/GoogleUtilities/Environment/Public/GoogleUtilities/GULSecureCoding.h
+++ b/GoogleUtilities/Environment/Public/GoogleUtilities/GULSecureCoding.h
@@ -25,7 +25,7 @@ NS_ASSUME_NONNULL_BEGIN
                                 fromData:(NSData *)data
                                    error:(NSError **)outError;
 
-+ (nullable id)unarchivedObjectOfClass:(Class)class
++ (nullable id)unarchivedObjectOfClass:(Class)aClass
                               fromData:(NSData *)data
                                  error:(NSError **)outError;
 

